src/explore-education-statistics-admin/src/services/permissionService.ts

```diff
@@ -16,6 +16,11 @@ export interface ReleaseStatusPermissions {
   canMarkHigherLevelReview: boolean;
   canMarkApproved: boolean;
 }
+export interface MethodologyStatusPermissions {
```

Comment:
Might be worth just extending from the ReleaseStatusPermissions type given they share the same implementation. We'd need to change ReleaseStatusPermissions from an interface to a type in order to avoid lint problems when doing this

i.e. 
```diff 
- export interface ReleaseStatusPermissions = { }
+ export type ReleaseStatusPermissions = {}

+ export type MethodologyStatusPermissions = ReleaseStatusPermissions
```

Reply:
I'm inclined to keep them separate, as release permissions aren't methodology permissions, even if they're identical. I'm all for DRY, but in this instance I think merging these two things is counter-intuitive. 

What do others think?